src/io_hid_keys.rs

```diff
@@ -0,0 +1,16 @@
+// exports from <IOKit/hid/IOHIDKeys.h>
+
+use libc::c_int;
+use types::IOOptionBits;
+
+pub type IOHIDReportType = c_int;
+pub const kIOHIDReportTypeInput: IOHIDReportType   = 0;
+pub const kIOHIDReportTypeOutput: IOHIDReportType  = 1;
+pub const kIOHIDReportTypeFeature: IOHIDReportType = 2;
+pub const kIOHIDReportTypeCount: IOHIDReportType   = 3;
+
+pub type IOHIDManagerOptions = IOOptionBits;
```

Comment:
This is defined in `IOHIDManager.h`, not `IOHIDKeys.h`. I think you may have missed this comment from my previous review:

> I see this definition in `IOKit/hid/IOHIDManager.h`. I may not have the latest SDK, so just let me know if this has changed since the version I'm looking at.

I'd like to keep this consistent with the structure that I explained in the ["contributing" section of the README](https://github.com/dcuddeback/iokit-sys#contributing).